data/migrations/20190311030207_notifications.js

```diff
@@ -1,15 +1,16 @@
 exports.up = function(knex, Promise) {
-  return knex.schema.createTable("watering", table => {
+  return knex.schema.createTable("notifications", table => {
     table.increments();
-    table.datetime("wateringTime");
+    table.datetime("notificationsTime");
```

Comment:
good change ... probably should be `notificationTime` since each row is a singular notification